skbio/stats/composition.py

```diff
@@ -838,6 +884,12 @@ def ancom(table, grouping,
     if (table.isnull()).any().any():
         raise ValueError('Cannot handle missing values in `table`.')
 
+    for percentile in percentiles:
+        if not 0.0 <= percentile <= 100.0:
+            raise ValueError('Percentiles must be in the range [0, 100], %r '
+                             'was provided.' % percentile)
+
```

Comment:
I think both are fine - the values are clearly defined in the resulting dataframe, so neither case would give the user incorrect information.